templates/hello_world_test.go

```diff
@@ -0,0 +1,7 @@
+package main
+
+import "testing"
+
+func TestHelloWorld(t *testing.T) {
+	t.Fatal("not implemented")
```

Comment:
I think this should be commented out by default. It's better to give people a clean pkg instead of a test that's fails. Could you comment this out? The function body should be like:

```
// t.Fatal("not implemented")
```